Explore.R

```diff
@@ -589,4 +589,111 @@ ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = Dominate.Strat
+
+ggplot(data = plot_data_team, aes_string(x = "Target", y = dependent_response_team, fill = "Team", group = "rank_order")) +
+  geom_bar(stat = "identity", position = "dodge") + 
+  facet_grid(. ~ SessionOrder) + 
+  guides(fill = FALSE) +
+  facet_grid(. ~ Dominate.Strategy) +
+  labs(y = y_label_team, x = x_label_team, title = title_response_team, fill = "Teams") 
+
+
+# Is there an interaction between the session order and the Target levels? -----
+plot_data_team <- team_data %>%
+  select(Target, SessionOrder, dependent_response_team) %>%
+  group_by(SessionOrder, Target) %>%
+  summarise(Average = mean(.data[[dependent_response_team]]), 
+            Stdv =sd(.data[[dependent_response_team]]), 
+            n = length(.data[[dependent_response_team]]), 
+            StEr = sd(.data[[dependent_response_team]]) / sqrt(length(.data[[dependent_response_team]])))
+
+ggplot(data = plot_data_team, aes(x = Target, y = Average, color = SessionOrder, shape = SessionOrder)) +
+  geom_point(size = 3) +
+  geom_line(aes(group=SessionOrder, color = SessionOrder)) + 
+  geom_errorbar(aes(ymin = Average - StEr, ymax = Average + StEr), width = 0.2) +
+  labs(y = y_label_team, x = x_label_team, title = title_response_team, color = "Session", shape = "Session")
+# Ind_Team collection rate seems higher than other Target levels, except when ordered in the 4th Session.
+# Ind_Team rate spreads out when compared to Ind or Team, seems like an odd patern
+
+plot_data_team <- team_data %>%
+  select(Target, SessionOrder, dependent_response_team) %>%
+  group_by(SessionOrder, Target) %>%
+  summarise(Average = mean(.data[[dependent_response_team]]), 
+            Stdv =sd(.data[[dependent_response_team]]), 
+            n = length(.data[[dependent_response_team]]), 
+            StEr = sd(.data[[dependent_response_team]]) / sqrt(length(.data[[dependent_response_team]])))
+
+ggplot(data = plot_data_ind, aes(x = Target, y = Average, color = SessionOrder, shape=SessionOrder)) +
+  geom_point(size = 3) +
+  geom_line(aes(group=SessionOrder, color = SessionOrder)) + 
+  geom_errorbar(aes(ymin = Average - StEr, ymax = Average + StEr), width = 0.2) +
+  labs(y = y_label_rate_team, x = x_label_team, title = title_response_team, fill = "Players")
+# Interaction plot showing team collection rate v. target by session order. It is interesting that sessions
+# 2 and 3 show a similar pattern of collection rate between targets. This may be a fact of the teams still
+# facing a learning curve. 
+
+ggplot(data = plot_data_ind, aes(x = SessionOrder, y = Average, color = Target, shape=Target)) +
+  geom_point(size = 3) +
+  geom_line(aes(group=Target, color = Target)) + 
+  geom_errorbar(aes(ymin = Average - StEr, ymax = Average + StEr), width = 0.2) +
+  labs(y = y_label_rate_team, x = x_label_teamOrder, title = title_response_team_rate_order, fill = "Players")
+# Interaction plot shoing team collection rate v. session order by Target. This shows how each target responded
```

Comment:
Make sure the interpretation is correct. The plot you were viewing when you wrote this may not be the correct plot.